README.md

```diff
@@ -167,7 +167,7 @@ code { font-family: 'Fira Code', monospace; }
 - ACE
 - CodeMirror (enable with `font-variant-ligatures: contextual;`)
 
-### Projects using Fira Code
+### Projects using the Fira Code
```

Comment:
Are you sure that we need "the" here?

Reply:
Fira code is a specific noun here. That's why "the" is needed.

Reply:
“Fira Code” is a proper name here. You don’t say “the Times New Roman” or “the Comic Sans”